src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  height: 408px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  padding-top: 32px;
+  padding-inline: 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    width: 166px;
+    height: 36px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $additional-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    color: $main-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $additional-text-color;
```

Comment:
The variable `$additional-text-color` is used here. Ensure that this variable is defined in your SCSS files, otherwise it will cause a compilation error.